src/rpm/builder.rs

```diff
@@ -591,13 +591,14 @@ impl RPMBuilder {
         // Lead is not important. just build it here
 
         let lead = Lead::new(&self.name);
-        let possible_compression_details = self.compressor.get_details();
 
         // Make the borrow checker happy
-        let is_zstd = matches!(self.compressor, Compressor::Zstd(_));
+        let is_zstd = matches!(self.compressor, CompressionType::Zstd);
```

Comment:
We don't need to make the borrow checker happy anymore ^.

Reply:
Done